packages/dicomImageLoader/src/imageLoader/wadors/metaData/getValue.ts

```diff
@@ -6,7 +8,11 @@
  * @param [defaultValue] - The default value to return if the element does not exist
  * @returns {*}
  */
-function getValue(element, index, defaultValue) {
+function getValue(
+  element: WADORSMetaDataElement,
+  index?: number,
+  defaultValue?: number | string
+): number[] | string[] | string | number {
```

Comment:
This is insufficient - there is also boolean, and there is no guarantee you won't get other types.  This should be unknown as a return type because it is hard to guess.  defaultValue should also be unknown, and should allow for the array types just as much as the simple types.

Reply:
We could add a generic type to the function rather than return unknown.

```
function getValue<ReturnType = unknown>(
   element: WADORSMetaDataElement,
   index?: number,
   defaultValue?: number | string
 ): ReturnType
```